lib/Mojo/Headers.pm

```diff
@@ -50,7 +56,22 @@ sub append {
   return $self->header($name => defined $old ? "$old, $value" : $value);
 }
 
-sub clone { $_[0]->new->from_hash($_[0]->to_hash(1)) }
+sub clone {
```

Comment:
Is the new clone() method faster than the old one or is there another reason for the rewrite?

Reply:
More than twice as fast.